moz_minions/minions.py

```diff
@@ -110,15 +118,10 @@ def _output(self, data):
         '''
         Default output to files with timestamp
         '''
-        try:
-            # TODO: parameter file path from config
-            # Should be a better way to do
-            timestamp = str(int(time.time()))
-            with open(self.output_file + "_" + timestamp, 'w') as oh:
-                oh.write(json.dumps(data))
-        except Exception as e:
-            print(e.message)
-            return False
+        timestamp = str(int(time.time()))
```

Comment:
what about:
    import time
    import datetime
    timestamp = what about:
    import time
    import datetime
    timestamp = datetime.datetime.fromtimestamp(time.time()).strftime('%Y%m%d%H%M%S')
Just to get a better timestamp (just a suggestion)